python/tk_multi_loader/loader_action_manager.py

```diff
@@ -276,26 +315,60 @@ def get_actions_for_folder(self, sg_data):
         :return: List of QAction instances.
         """
 
-        actions = []
+        qt_actions = []
+
+        # If the selection is empty, we skip this and append default actions
```

Comment:
I think your comment might be out of date. It looks like if the selection is not empty, we add some custom actions and then the default actions.